ast.ts

```diff
@@ -37,6 +39,8 @@ export type Expr<A> =
   | {  a?: A, tag: "lookup", obj: Expr<A>, field: string }
   | {  a?: A, tag: "method-call", obj: Expr<A>, method: string, arguments: Array<Expr<A>> }
   | {  a?: A, tag: "construct", name: string }
+  | {  a?: A, tag: "list_expr", contents: Array<Expr<A>> }
```

Comment:
Any reason for breaking with the `kebab-case` style in favor of `snake_case`? Sorry I missed that last night.